audit/result.go

```diff
@@ -74,25 +74,21 @@ func (r *Result) Unmarshal(data []byte) error {
 
 	err = cID.ReadFromV2(*cidV2)
 	if err != nil {
-		return fmt.Errorf("could not convert V2 container ID: %w", err)
+		return fmt.Errorf("invalid container ID: %w", err)
 	}
 
-	var (
-		oID   oid.ID
-		oidV2 refs.ObjectID
-	)
-
-	for _, oidV2 = range r.v2.GetPassSG() {
+	var oID oid.ID
+	for i, oidV2 := range r.v2.GetPassSG() {
 		err = oID.ReadFromV2(oidV2)
 		if err != nil {
-			return fmt.Errorf("invalid passed storage group ID: %w", err)
+			return fmt.Errorf("invalid passed storage group ID #%d: %w", i, err)
```

Comment:
we have an exact ID we could not read here, why not add it? same below

Reply:
cuz it can be few MB len. We could make a cut for overlen values `abc...def` but code'd become more complex